app/Http/Controllers/SonController.php

```diff
@@ -56,7 +58,7 @@ public function index(Son $son)
     public function create()
     {
         $form_field     = SonFormField::first();
-        $bc1             = $this->bc1;
+        $bread_crumbs             = $this->bread_crumbs;
```

Comment:
If you're going to use your fancy formatting... ;)
```suggestion
        $bread_crumbs   = $this->bread_crumbs;
```